luthier/config/src/metrics.lua

```diff
@@ -86,11 +91,18 @@ return {
         druidMetric = "NO2"
     },
     averageCOPerDay = {
-        maker = "aggregateAveragebyDay",
+        maker = "aggregateAverageDAY",
         dependencies = {"COM"}
     },
     averageNO2PerDay = {
-        maker = "aggregateAveragebyDay",
+        maker = "aggregateAverageDAY",
         dependencies = {"NO2M"}
     }
 }
+
+for metric_name, metric_content in pairs(metrics) do
```

Comment:
This belongs in the defaulting file.

Reply:
it looks like a defaulting but I am putting it there to highlight the fact that it is a mandatory attribute the user should specify (as is for all of the `type`s in other configs), so it is more of a short hand instead of making it looks like an optional value, where defaultingUtil can help populating.

Reply:
Why not? Do we expect people to *ever* use anything other than the default? We should document that the field exists of course, but seeing as how all the interesting customization is happening in the makers, and not in the metric, I don't see why this shouldn't be a default. For most people, this type field is going to be ceremony if they have to specify it. And we want as little ceremony as possible.